player/public/coreplugins/org.ekstep.ecmlrenderer-1.0/renderer/plugin.js

```diff
@@ -184,6 +184,7 @@ org.ekstep.contentrenderer.baseLauncher.extend({
         content.canvasId = dataObj.canvasId;
         EkstepRendererAPI.setRenderer(instance);
         Renderer.theme = new ThemePlugin(content);
+        qspatch.setDefaultFontSize(content);
```

Comment:
This should be just before 
line 202: qspatch.handleAssetUrl();

Reply:
Done